### PR TITLE
Adds additional units and route to prescriptions

### LINKED ISSUE #8059: Prescription - improvement to dropdown
**Is your feature request related to a problem? Please describe.**
To improve the prescription module, add the following

**Describe the solution you'd like**

- [ ] In dosage unit dropdown add "mcg" and "units"
- [ ] In route add "Sublingual"

### PR DESCRIPTION
**Pull request by 0hmX**
## Required Backends

- https://github.com/coronasafe/care/pull/2290

## Proposed Changes

- Fixes #8059 
- In dosage units dropdown added "mcg" and "unit(s)" units
- In routes added "Sublingual"

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Medicine/models.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { PerformedByModel } from "../HCX/misc";
 import { PRESCRIPTION_ROUTES } from "./CreatePrescriptionForm";
 
 export const DOSAGE_UNITS = [
+  "msg",
```
Comment (nihal467) [inline src/Components/Medicine/models.ts]:
```suggestion
  "mcg",
```

**Comment by nihal467** [comment]
image
image

- [ ] the PR is not allowing the user to create a medicine in mcg and unit dose or the newly added route is not selectable. Make necessary backend changes and mention the PR along with this PR for review purposes.

**Comment by 0hmX** [comment]
thanks for the review @nihal467 and remark i will make the needed changes

**Comment by aparnacoronasafe** [comment]
@nihal467 in dosage "unit" also needs to be added in the dropdown. As per the issue

**Comment by aparnacoronasafe** [comment]
@cu8code please bring this to a closure. Hoping to see this go to production on Monday

**Comment by nihal467** [comment]
@cu8code Due to inactivity I am closing this non-functional PR and assigning this issue to @rithviknishad  its been escalated to P1 priority

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**